protocol/x/clob/keeper/keeper.go

```diff
@@ -54,8 +53,7 @@ type (
 		streamingManager         streamingtypes.FullNodeStreamingManager
 		finalizeBlockEventStager finalizeblock.EventStager[*types.ClobStagedFinalizeBlockEvent]
 
-		initialized         *atomic.Bool
-		memStoreInitialized *atomic.Bool
+		initialized *atomic.Bool
```

Comment:
suggestion: rename to `inMemoryStructsInitialized` or sth similar to disambiguate with Mem Store